### ISSUE TITLE
Have several tags for a single room?

### ISSUE THREAD

**Issue by elinorbgr** [turt2live/matrix-tag-manager#3]
When using tags for sorting my rooms, I have several rooms that would meaningfully have more than one tag. It'd be great if this tool could support it!

... Though I'll admit riot's UI is not really designed to handle this. :(

**Comment by turt2live** [comment]
Riot not supporting this well was a leading factor into not supporting this fwiw

**Comment by elinorbgr** [comment]
Yeah, I guess this should rather wait to see how https://github.com/vector-im/riot-web/issues/8609 goes, or if there are new clients that support it.

**Comment by elinorbgr** [comment]
For what it's worth, [nheko-reborn](https://github.com/Nheko-Reborn/nheko) does support tags-as-filters and multiple tags on the same room.

**Comment by MTRNord** [comment]
I would like that too as I have rooms that fit in multiple categories and I want some to be in a custom tag as well as in my favs. Also the matrix proto obviously supports it and Riot is able to display it correct.
